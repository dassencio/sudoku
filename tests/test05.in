2 4
3 _ _ _ _ 7 5 _
_ 2 _ _ 3 _ _ _
_ 1 2 7 _ _ _ _
_ _ 6 4 _ 2 _ _
_ _ 1 _ 8 6 _ _
_ _ _ _ 4 5 1 _
_ _ _ 6 _ _ 7 _
_ 7 8 _ _ _ _ 5
