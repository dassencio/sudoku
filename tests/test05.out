3 8 4 1 2 7 5 6
6 2 7 5 3 1 4 8
8 1 2 7 5 3 6 4
5 3 6 4 7 2 8 1
4 5 1 2 8 6 3 7
7 6 3 8 4 5 1 2
2 4 5 6 1 8 7 3
1 7 8 3 6 4 2 5
